sdks/python/test-suites/tox/py38/build.gradle

```diff
@@ -32,4 +32,4 @@ check.dependsOn formatter
 apply from: "../common.gradle"
 
 // TODO(BEAM-8954): Remove this once tox uses isolated builds.
-testPy38Cython.mustRunAfter testPython38, testPy38Cloud
+testPy38Cython.mustRunAfter testPython38, python38CloudCoverage
```

Comment:
I think you meant `testPy38CloudCoverage`

Reply:
Thanks! Fixed